service/topology_coordinator.cc

```diff
@@ -752,6 +753,87 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
         case global_topology_request::cleanup:
             co_await start_cleanup_on_dirty_nodes(std::move(guard), true);
             break;
+        case global_topology_request::keyspace_rf_change: {
+            while (true) {
+                auto tmptr = get_token_metadata_ptr();
+                sstring ks_name = *_topo_sm._topology.new_keyspace_rf_change_ks_name;
+                std::unordered_map<sstring, sstring> new_rf_per_dc = *_topo_sm._topology.new_keyspace_rf_change_rf_per_dc;
+                std::map<sstring, sstring> new_rf_per_dc_map{new_rf_per_dc.begin(), new_rf_per_dc.end()};
+                sstring req_uuid = *_topo_sm._topology.global_request_id;
+                std::vector<canonical_mutation> updates;
+                sstring error;
+                auto& ks = _db.find_keyspace(ks_name);
+                tablet_replica_calculation_result new_data{locator::tablet_map{8}, tablet_replica_calculation_status::success};
```

Comment:
8?

Please don't use meaningless constants.
It looks like you only need the status outside the for loop, so please just keep that in a variable,
not the whole `tablet_replica_calculation_result`.

Reply:
ping.

Dropping `tablet_replica_calculation_status` and `tablet_replica_calculation_result` will greatly simplify this path.

Cc @KrzaQ

Reply:
`locator::tablet_map` needs to be initialized with something, I took 8 as a random param, i.e. initial tablets count, but I agree, it's not intuitive. As I still need to init it with something, I chose 0.
I removed the status altogether, and yes, it simplified the logic, thanks.